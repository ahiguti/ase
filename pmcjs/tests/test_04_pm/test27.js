
use('Perl');
Perl.Use('Net::servent');
var servent = Perl.Functions['Net::servent'];
var s = servent.getservbyname('ftp');
print("name=" + s.name());
print("port=" + s.port());

