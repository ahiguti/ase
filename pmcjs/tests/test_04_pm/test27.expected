name=ftp
port=21
